src/PrestaShopBundle/Resources/config/services/bundle/api.yml

```diff
@@ -23,3 +23,7 @@ services:
 
   prestashop.core.api.stock.movements_collection:
     class: PrestaShopBundle\Api\Stock\MovementsCollection
+
+  PrestaShopBundle\ApiPlatform\Normalizer\:
```

Comment:
This should be in the oauth.yml file

Reply:
I moved it back yesterday because it breaks the endpoint if it's not here.